src/main.c

```diff
@@ -173,7 +223,16 @@ void main(void)
 {
     vector_init(&tests_to_run, kernel_thunk_table_size);
     vector_init(&tests_exclude, 20);
-    if (!open_output_file("D:\\kernel_tests.log")) {
+    load_name_file("D:\\name.txt");
```

Comment:
When `load_name_file()` is changed into returning the `name_log` value, that variable no longer has to be global-scoped and thus can become a local variable (which can be declared in this same line where it would be assigned the return value from `load_name_file()`).